wgkex/worker/app.py

```diff
@@ -21,6 +30,11 @@ def main():
     domains = config.load_config().get("domains")
     if not domains:
         raise DomainsNotInConfig("Could not locate domains in configuration.")
+    clean_up_threads = []
+    for domain in domains:
+        thread = threading.Thread(target=clean_up_worker, args=(domain[2:],))
```

Comment:
You probably want to change the domain argument here ;)

Reply:
Yep :D. I was just testing it. Thanks for the hint.